examples/gpt2_e2e.py

```diff
@@ -0,0 +1,120 @@
+import os
+import numpy
+from transformers import AutoConfig
+from onnxruntime_customops import mytorch as torch
```

Comment:
I would not add an alias `torch`, it is difficult to know what's coming from torch or from customops.